setup.py

```diff
@@ -27,7 +27,7 @@ def read(fname):
     install_requires=(
         'Django>=1.5',
         'easy-thumbnails>=1.0',
-        'django-mptt>=0.6',
+        'django-mptt>=0.6,<0.8',
```

Comment:
You can't express versioned dependency on Django, and until filer supports django<<1.8 we can't include mptt>=0.8 in dependencies.
The major issues is with tests, in which you have to pin mptt in every Django 1.6/1.7 environment, which seems a huge burden.